### ISSUE TITLE
Refresh "OpenSSF Best Practices" questionaires

### ISSUE THREAD

**Issue by viccuad** [kubewarden/kubewarden-controller#789]
Some time has passed since we filled the https://www.bestpractices.dev questionaires.

## Acceptance criteria

- Revisit https://www.bestpractices.dev/en/projects?q=kubewarden & https://www.bestpractices.dev/en/projects?q=policy-server questionnaires.
- See if the policy-server one can be folded into the kubewarden project.

**Comment by jvanz** [comment]
I guess that I'm the owner of the kwctl repository because I've created the project for it. I cannot found an option to move this for the Kubewarden user. Therefore, I think we need to recreate it using the right user. As I do not have access to the user, I cannot do it. Once we do that, I'll delete the project in my user.

**Comment by jvanz** [comment]
I've deleted the kwctl project. Thus, we can recreate it using the Kubewarden user.

**Comment by viccuad** [comment]
I can't recreate it, the owner gets set as my personal name instead of "Kubewarden".

**Comment by flavio** [comment]
I guess we can move this card under "in progress" now that we have access to the OpenSFF website as "kubewarden"

**Comment by viccuad** [comment]
Recreated kwctl, refreshed policy-server, audit-scanner, kubewarden-controller: https://www.bestpractices.dev/en/users/23101

Once the linked PR is merged this is done.
